src/components/Composer/index.tsx

```diff
@@ -237,39 +231,24 @@ function Composer(
     );
 
     /**
-     * Check the current scrollHeight of the textarea (minus any padding) and
-     * divide by line height to get the total number of rows for the textarea.
+     * Check the current text input width and update the state with the new width.
      */
-    const updateNumberOfLines = useCallback(() => {
+    const updateTextInputWidth = useCallback(() => {
```

Comment:
The naming seems confusing here, since we also update the height

Reply:
I removed height updates, so now we are only updating width. I will investigate if we can remove this whole function